docs/website/blog/2023-11-01-dlt-dagster.md

```diff
@@ -0,0 +1,421 @@
+---
+slug: dlt-dagster
+title: "Orchestrating unstructured data pipeline with dagster and dlt."
+image: https://d1ice69yfovmhk.cloudfront.net/images/dlt-dagster_overview.jpg
+authors:
+    name: Zaeem Athar
+    title: Junior Data Engineer
+    url: https://github.com/zem360
+    image_url: https://images.ctfassets.net/c4lg2g5jju60/5tZn4cCBIesUYid17g226X/a044d2d471ebd466db32f7868d5c0cc8/Zaeem.jpg?w=400&h=400&q=50&fm=webp
+tags: [Dagster, dlt, Asset Factory, Unstructured Data]
+---
+:::info
+TL;DR: In this blog post, we'll build data piplines using dlt and orchestrate them using Dagster.
+:::
+
+`dlt` is an open-source Python library that allows you to declaratively load messy data sources into well-structured tables or datasets, through automatic schema inference and evolution. It simplifies building data pipelines by providing functionality to support the entire extract and load process.
+
+It does so in a scalable way, enabling you to run it on both micro workers or in highly parallelized setups. dlt also offers robustness on extraction by providing state management for incremental extraction, drop-in requests replacement with retries, and many other helpers for common and uncommon extraction cases.
+
+To start with `dlt`, you can install it using pip: `pip install dlt`. Afterward, import `dlt` in your Python script and start building your data pipeline. There's no need to start any backends or containers. 
+
+## **Project Overview:**
+
+In this example, we will ingest GitHub issue data from a repository and store the data in BigQuery. We will use `dlt` to create a data pipeline and orchestrate it using dagster. 
+
+Initially, we will start by creating a simple data pipeline using `dlt`.  We will then orchestrate the pipeline using dagster. Finally, we will add more features to this pipeline by using the dlt schema evolution and dagster asset metadata to educate the users about their data pipeline.
```

Comment:
capitalize Dagster. somewhere you have it with a capital letter, somewhere with a small letter. maintain a consistent writing style throughout the text